exports/sharedAbis.json

```diff
@@ -0,0 +1,224 @@
+{
+    "ERC20": [
```

Comment:
Hmmm, why do we need this json? ERC20 is provided in the openzeppelin package, why do we need JSON of its ABI?

Reply:
The idea here is that everyone who needs the contract information, especially the frontend guys, have all relevant information available in one place.

Reply:
Well, it can be stored on frontend only, but Ok...

Reply:
> Well, it can be stored on frontend only, but Ok...

I don't understand this comment. What would be stored on the frontend?